### DISCUSSION TITLE
SFrame move scroll

### DISCUSSION THREAD

**Discussion by PraryDogy** [Saadmairaj/tkmacosx#42] [General]
Hi, has tkmacosx.SFrame any option to force move scroll to the bottom or on any coordinates, for example?

**Comment by Saadmairaj** [comment]
Hi

The SFrame widget uses canvas as its main frame and you can access it with `sframe['canvas']` through that you can access all features of the canvas one of which is `yview_moveto(...)`. See the Tkinter canvas documentation for all the methods and configurations.

Here is an example of scrolling to the bottom of the frame with the click of a button.

```
from tkinter import Tk
from tkmacosx import SFrame, Button

root = Tk()
root.geometry('350x400')
sframe = SFrame(root, bg='#E69A8D', scrollbarwidth=3)
sframe.pack(pady=20)

for i in range(50):
    Button(sframe, text='Button %s' % i).pack()

button_to_scoll = Button(root, text="Scroll to bottom",
                         command=lambda: sframe['canvas'].yview_moveto('1.0'))
button_to_scoll.pack()

root.mainloop()
```

One thing to keep in mind though, is that calling `sframe['canvas'].yview_moveto('1.0')` before the mainloop won't work unless you update the window with either `root.update()` or `root.update_idletasks()` before.
